osu.Game/Rulesets/Difficulty/DifficultyAttributes.cs

```diff
@@ -106,8 +106,8 @@ public virtual void FromDatabaseAttributes(IReadOnlyDictionary<int, double> valu
             MaxCombo = (int)values[ATTRIB_ID_MAX_COMBO];
 
             // Temporarily allow these attributes to not exist so as to not block releases of server-side components while these attributes aren't populated/used yet.
-            LegacyAccuracyScore = (int)values.GetValueOrDefault(ATTRIB_ID_LEGACY_ACCURACY_SCORE);
-            LegacyComboScore = (int)values.GetValueOrDefault(ATTRIB_ID_LEGACY_COMBO_SCORE);
+            LegacyAccuracyScore = (long)values.GetValueOrDefault(ATTRIB_ID_LEGACY_ACCURACY_SCORE);
+            LegacyComboScore = (long)values.GetValueOrDefault(ATTRIB_ID_LEGACY_COMBO_SCORE);
```

Comment:
I'm tackling this from a different route - moving these attributes into a new table that has the precise data types required. Not because of this comment/PR, but because we recently noticed the prod table is already 117GB without these attributes even being populated yet.

Reply:
To elaborate on the above: the existing table stores per-mod lookups, but for the new legacy score values, they do not need to be stored per mod. So if we were storing to the existing table, special cases would be required for these lookups, or they would need to be stored 50ish times per map for every mod combo (which is a huge waste).

So I proposed a separate table.